pub/errors/processor.php

```diff
@@ -265,7 +265,8 @@ public function getHostUrl()
             $host = 'localhost';
         }
 
-        $isSecure = (!empty($_SERVER['HTTPS'])) && ($_SERVER['HTTPS'] != 'off');
+        // HTTP_X_FORWARDED_PROTO to check whether a webserver using HTTP is behind a load balancer serving HTTPS
```

Comment:
I think comment is not needed there, it's pretty obvious for what this header is needed.